cwms-servers/CWMS_HOME/bin/stopLoginServerLogWatcher

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# stopAccessLogMonitor.sh
+
+# Read the saved PID
+pid=$(cat "$CWMS_HOME/dated/logs/loginServer/accessLogMonitor.pid")
```

Comment:
what if this file is gets deleted?

Reply:
Then the log file process would keep going (it won't terminate the StopCWMS process). The alternative is to `pgrep` and find all python process running `script_path="$CWMS_HOME/bin/logWatcher.py"`. That assumes that the script is only getting run from BootCWMS though.

Reply:
This should be done, process management needs to be more robust.

Reply:
Added a follow-up issue to handle the robust exit: https://github.com/USACE-WaterManagement/cwms-logging/issues/3